src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMessage.cs

```diff
@@ -46,6 +46,12 @@ public abstract class CoseMessage
         /// <value>A collection of unprotected header parameters associated to this message.</value>
         public CoseHeaderMap UnprotectedHeaders => _unprotectedHeaders;
 
+        /// <summary>
+        /// Gets the raw bytes of the protected header parameters associated to this message.
```

Comment:
```suggestion
        /// Gets the raw bytes of the protected header parameters associated with this message.
```

Language nerd time: As far as I can tell, Latin languages use "associate to" and Germanic languages use "associate with".  While English usually accepts both forms, American English doesn't (still AFAICT) accept "associate to".

Reply:
Ok, there are other places where we use "associated to", I will change them as well since this PR is not cumbersome.